packages/repository/test/unit/decorator/model-and-relation.ts

```diff
@@ -70,6 +85,8 @@ describe('model decorator', () => {
 
     // Validates that property no longer requires a parameter
     @property() isShipped: boolean;
+
+    @array(Product) items: Product[];
```

Comment:
It might be better to name it as `@property.array` to be consistent with other decorators we have, such as:
- `@inject.getter`
- `@param.path.string`